docs/details/victoria-metrics.md

```diff
@@ -0,0 +1,33 @@
+# VictoriaMetrics
+
+[VictoriaMetrics](https://victoriametrics.github.io/) is a third-party monitoring solution and time-series database that replaced Prometheus in [PMM 2.12.0](../release-notes/2.12.0.md).
+
+## Remapped targets for direct Prometheus paths
+
+Direct Prometheus paths return structured information directly from Prometheus, bypassing the PMM application.
+
+They are accessed by requesting a URL of the form `<PMM SERVER URL>/prometheus/<token>`.
```

Comment:
```suggestion
They are accessed by requesting a URL of the form `<PMM SERVER URL>/prometheus/<PATH>`.
```

Reply:
'token'  usually used for some security-related topics and it's Key - Secret part